tonic/src/transport/channel/service/tls.rs

```diff
@@ -34,7 +34,10 @@ impl TlsConnector {
 
         #[cfg(feature = "tls-native-roots")]
         if with_native_roots {
-            roots.add_parsable_certificates(rustls_native_certs::load_native_certs()?);
+            let rustls_native_certs::CertificateResult { certs, errors, .. } =
```

Comment:
Suggest yielding an error if `certs.is_empty()`, at least.

Reply:
Updated to return a error when the certs is empty.